testsuite/testcases/src/test/cases/java/java-2.input

```diff
@@ -1,6 +1,6 @@
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
- * JFlex 1.4.1                                                             *
- * Copyright (C) 1998-2004  Gerwin Klein <lsf@jflex.de>                    *
+ * JFlex 1.7.0                                                             *
```

Comment:
That change might lead to confusion down the track. The test input is just a file picked from 1.4.1 back then, i.e. updating the version number in it will lead us down the wrong track if we ever want to know where it came from. Are you happy for me to amend/revert this bit?